src/main/java/hw_9/task1/Earth.java

```diff
@@ -0,0 +1,11 @@
+package hw_9.task1;
+
+public class Earth implements Planet {
+    String name = "Earth";
```

Comment:
where is access modifier? And probably it is a constant? Fix in all places please

Reply:
fixed